README.md

```diff
@@ -1,84 +1,95 @@
  - Browser & Workers environment compatibility
  - Demonstrates good performance in benchmarks
 
 ```
-EventEmitterHeatUp x 3,728,965 ops/sec \302\2610.68% (60 runs sampled)
-EventEmitter x 2,822,904 ops/sec \302\2610.74% (63 runs sampled)
-EventEmitter2 x 7,251,227 ops/sec \302\2610.55% (58 runs sampled)
-EventEmitter2 (wild) x 3,220,268 ops/sec \302\2610.44% (65 runs sampled)
+Platform: win32, x64, 15267MB
+Node version: v13.11.0
+CPU: 4 x AMD Ryzen 3 2200U with Radeon Vega Mobile Gfx @ 2495MHz
+----------------------------------------------------------------
+EventEmitterHeatUp x 3,167,076 ops/sec ±3.17% (59 runs sampled)
+EventEmitter x 3,190,460 ops/sec ±3.20% (66 runs sampled)
+EventEmitter2 x 11,278,456 ops/sec ±4.26% (60 runs sampled)
+EventEmitter2 (wild) x 4,620,369 ops/sec ±4.46% (61 runs sampled)
+EventEmitter3 x 10,309,717 ops/sec ±3.89% (64 runs sampled)
+
 Fastest is EventEmitter2
 ```
 
 ### Differences (Non-breaking, compatible with existing EventEmitter)
 
- - The EventEmitter2 constructor takes an optional configuration object.
- 
+ - The EventEmitter2 constructor takes an optional configuration object with the following default values:
 ```javascript
-var EventEmitter2 = require('eventemitter2').EventEmitter2;
-var server = new EventEmitter2({
-
-  //
-  // set this to `true` to use wildcards. It defaults to `false`.
-  //
-  wildcard: true,
-
-  //
-  // the delimiter used to segment namespaces, defaults to `.`.
-  //
-  delimiter: '::', 
-  
-  //
-  // set this to `true` if you want to emit the newListener event. The default value is `false`.
-  //
+var EventEmitter2 = require('eventemitter2');
+var emitter = new EventEmitter2({
+
+  // set this to `true` to use wildcards
+  wildcard: false,
+
+  // the delimiter used to segment namespaces
+  delimiter: '.', 
+
+  // set this to `true` if you want to emit the newListener event
   newListener: false, 
-  
-  //
-  // set this to `true` if you want to emit the removeListener event. The default value is `false`.
-  //
+
+  // set this to `true` if you want to emit the removeListener event
   removeListener: false, 
 
-  //
-  // the maximum amount of listeners that can be assigned to an event, default 10.
-  //
-  maxListeners: 20,
-  
-  //
-  // show event name in memory leak message when more than maximum amount of listeners is assigned, default false
-  //
+  // the maximum amount of listeners that can be assigned to an event
+  maxListeners: 10,
+
+  // show event name in memory leak message when more than maximum amount of listeners is assigned
   verboseMemoryLeak: false,
 
-  //
   // disable throwing uncaughtException if an error event is emitted and it has no listeners
-  //
   ignoreErrors: false
 });
 ```
 
  - Getting the actual event that fired.
 
 ```javascript
-server.on('foo.*', function(value1, value2) {
```

Comment:
I think it's good to have the values in this example since it showcases that we support emitting multiple values at once. Maybe those values should be added to the `.emit()` calls?

Reply:
Ok. Just added this.